core/state_processor_test.go

```diff
@@ -616,3 +625,902 @@ func getParentBlockHash(statedb *state.StateDB, number uint64) common.Hash {
 	binary.BigEndian.PutUint64(key[24:], ringIndex)
 	return statedb.GetState(params.HistoryStorageAddress, key)
 }
+
+var testKaustinenLikeChainConfig = &params.ChainConfig{
+	ChainID:                 big.NewInt(69420),
+	HomesteadBlock:          big.NewInt(0),
+	EIP150Block:             big.NewInt(0),
+	EIP155Block:             big.NewInt(0),
+	EIP158Block:             big.NewInt(0),
+	ByzantiumBlock:          big.NewInt(0),
+	ConstantinopleBlock:     big.NewInt(0),
+	PetersburgBlock:         big.NewInt(0),
+	IstanbulBlock:           big.NewInt(0),
+	MuirGlacierBlock:        big.NewInt(0),
+	BerlinBlock:             big.NewInt(0),
+	LondonBlock:             big.NewInt(0),
+	Ethash:                  new(params.EthashConfig),
+	ShanghaiTime:            u64(0),
+	VerkleTime:              u64(0),
+	TerminalTotalDifficulty: common.Big0,
+}
+
+// TestProcessVerkleInvalidContractCreation checks for several modes of contract creation failures
+func TestProcessVerkleInvalidContractCreation(t *testing.T) {
```

Comment:
Isn't it better to add all these to a file `verkle_processing_test.go`, or `verkle_state_processor_test.go`?